zuri_net/src/proto/packet/mob_effect.rs

```diff
@@ -62,4 +62,5 @@ pub enum MobEffectType {
     Levitation,
     FatalPoison,
     ConduitPower,
+    SlowFalling,
```

Comment:
Did we never have this before? It's definitely not new in these updates

Reply:
I basically just followed gophertunnel's commit history since the latest version we supported. Based off of that, it seems to have been added to GT here: https://github.com/Sandertv/gophertunnel/commit/9008a06a35c2dd0f8a72e7ce6ac4666891825dce